ingesters/s3.md

```diff
@@ -0,0 +1,113 @@
+
+## Basic Configuration
+
+The Gravwell S3 ingester can consume from multiple buckets at once and will track its progress using a state file so that you can start and stop the ingester at will.  By default, the ingester will scan S3 buckets every minute looking for changes and ingest new objects that are discovered.  The main configuration for the Gravwell S3 ingester requires a `[Global]` section that specifies all of the usual configuration parameters which tell the ingester how to connect to Gravwell indexers.  The ingester also supports the full suite of [custom preprocessors](https://docs.gravwell.io/#!ingesters/preprocessors/preprocessors.md), [time format overrides](https://docs.gravwell.io/#!ingesters/ingesters.md#Time_Parsing_Overrides),  and [custom time format specifications](https://docs.gravwell.io/#!ingesters/customtime/customtime.md).
+
+The S3 ingester relies on a standard configuration file located at `/opt/gravwell/etc/s3.conf` by default, but also supports configuration overlays in the `/opt/gravwell/etc/s3.conf.d/` directory.  Remember to ensure that all configuration overlays have the `.conf` file extension and are readable by the `gravwell` user and/or `gravwell` group.
+
+A single S3 ingester can specify multiple buckets to consume from.  Multiple buckets are specified by creating multiple `[Bucket "name"]` stanzas in either the main `s3.conf` file or in overlay configuration files.
+
+Note: We HIGHLY recommend creating a dedicated S3 IAM user for the Gravwell ingester.  It's never a good idea to use privileged credentials for dedicated applications like data ingestion.
+
+### Bucket Configuration
+
+Each bucket configuration supports the following configuration options:
+
+| Configuration Parameter | Required      | Default Value | Description |
+|-------------------------|---------------|---------------|-------------|
+| ID                      | *YES*         |               | AWS IAM User ID to authenticate with |
+| Secret                  | *YES*         |               | AWS IAM User secret to authenticate with |
+| Region                  | *YES*         |               | AWS region the S3 bucket is registered in (Example: `us-east-2`) |
+| Bucket-ARN              | *YES*         |               | AWS S3 bucket resource name (Example: `arn:aws:s3:::aws-cloudtrail-logs-475058115300-aabbccddee`) |
+| Tag-Name                | *YES*         |               | Specify the Gravwell tag name data ingested from the bucket |
```

Comment:
Specify the Gravwell tag name "to be applied to" ... ?

Reply:
used suggestion